lib/run-datadog.sh

```diff
@@ -223,7 +223,67 @@ monit_datadog() {
   done
 }
 
+enable_apm_ssi() {
+  echo "Enabling Single Step APM Instrumentation"
+
+  PIP_CMD=""
+  PYTHON_BUILDPACK_DIR=""
+
+  # add all language buildpack bin folders to the PATH
+  for dir in $DEPS_DIR/*/; do
+    dir="${dir%/}" # remove trailing slash
+    if grep -q -E "name: (python|ruby|go|nodejs|java)" "$dir/config.yml" >/dev/null 2>&1; then
```

Comment:
Could we use `staging_info.yml` to detect the buildpack?

Reply:
We could, but I believe it's safer to use the `config.yml` as it is explicitly documented in the buildpacks contract.
See https://docs.cloudfoundry.org/buildpacks/custom.html